### PR TITLE
time: add feature to have additional networks allowed

### PR DESCRIPTION
**Pull request by MarbolanGos**
Hello,

Here is a little patch I had to setup. Maybe this is not the best way to write it but it works. In my case, I had to add some routed networks to my chrony-server which were allowed to get access to the time service.
Before this patch, the networks were all extracted from the NIC interfaces but this was not enough.

This solves this issue: https://github.com/bluebanquise/bluebanquise/issues/553

### CODE REVIEW

File: roles/core/time/templates/chrony.conf.j2 [hunk 1]
```diff
@@ -57,6 +57,11 @@ rtcsync
 allow {{ networks[nic.network]['subnet'] }}/{{ networks[nic.network]['prefix'] }}
     {% endif %}
   {% endfor %}
+  {% if time_additional_networks_allowed is defined and time_additional_networks_allowed is not none %}
```
[the remark this answers is not in the document]
Comment (oxedions) [reply roles/core/time/templates/chrony.conf.j2:60]: Dear @MarbolanGos

Any update on this PR ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
   - pxe_stack: fix issues with hostname not set during kickstart on RHEL 8.3 (#522)
   - set_hostname: add fqdn capability (#543)
   - time: allow to set sysconfig OPTIONS for chronyd (#552)
+  - time: allow to additional networks for server to reply (#555)
```
Comment (btravouillon) [inline CHANGELOG.md:18]: There should be a typo here: `allow to _verb_ additional` (configure, define, add, your choice :smiley:)

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Comment by oxedions** [comment]
Same here. Thanks a lot 😊

**Pull request merged**